scripts/index.js

```diff
@@ -31,10 +29,31 @@ const normalizeResponse = response => {
 
 const handleSupported = () =>
   fetch(instagramURL)
+    .then(response => {
+      if (!response.ok) {
+        throw new Error('⛔️ Something went wrong');
+      }
+
+      return response;
+    })
     .then(response => validateResponse(response))
     .then(response => normalizeResponse(response))
-    .then(payload => Meta(payload))
-    .catch(error => console.info(error));
+    .then(payload => {
+      import('./Meta/index.js').then(module => module.default(payload));
+      import('./Meta/parallax.js').then(module =>
+        document.addEventListener('mousemove', module.default)
+      );
+    })
+    .catch(error => {
+      document.getElementById('message').textContent = `⛔️ ${error}`;
+      document.getElementById('context').innerHTML =
+        'The <a href="https://addons.mozilla.org/en-US/firefox/addon/facebook-container/">\
+        Mozilla Facebook Container Extension</a> prevents loading the Instagram\
+        API from the private/unsupported endpoint that is currently being used.\
+        This is a known issue and a solution using the\
+        <a href="https://developers.facebook.com/docs/instagram-basic-display-api/reference/media">\
+        Instagram Basic Display API</a> is currently in development.';
```

Comment:
do you want to catch the specific error that is thrown when using Mozilla rather than showing this as a blanket error message for all errors? What if there is a different error unrelated to Mozilla?

Reply:
Unfortunately the only error that I get is the one you see in the screenshot (in the title).

I decided to put this blanket error for now because I know that this specific issue is happening and wanted to point it out as a most-likely culprit. … but now I'm wondering if I can detect whether the extension is active/enabled and display that text conditionally. I'm gonna give that a shot today.

Reply:
That makes sense. I approve this PR then!

Reply:
It looks like extension detection is not possible… which is a good thing because it means the API is secure. But You got me thinking about this blanket error context and I've decided it's best to only display it when `TypeError: NetworkError when attempting to fetch resource.` is the error returned. It could still be a false positive until I set up my proxy, but I think that's acceptable in the meantime. Once the proxy is set up I'll remove the condition/context.